docs/running-on-yarn.md

```diff
@@ -426,8 +426,10 @@ To use a custom metrics.properties for the application master and executors, upd
 Standard Kerberos support in Spark is covered in the [Security](security.html#kerberos) page.
 
 In YARN mode, when accessing Hadoop file systems, aside from the service hosting the user's home
```

Comment:
No your fault, but this doesn't seem accurate given the code, which doesn't seem to look at the home directory at all.